core/templates/dev/head/services/explorationContextService.js

```diff
@@ -49,7 +50,8 @@ oppia.factory('explorationContextService', [
       },
       // Returns a string representing the context of the current page.
       // This is either PAGE_CONTEXT.EDITOR or PAGE_CONTEXT.LEARNER.
-      // If the current page is not one of these, an error is raised.
+      // If the current page is not one in either EDITOR or LEARNER then
+      // return PAGE_CONTEXT.OTHER
```

Comment:
Ah, that change was needed.
Checkout https://github.com/oppia/oppia/issues/1504#issuecomment-192587136 comment of mine, I think you will understand it.
If it does not help then I will be happy to help.